src/test/java/edu/ksu/canvas/attendance/controller/CourseConfigurationControllerITest.java

```diff
@@ -92,19 +92,19 @@ public void classSetup_existingSectionId_HappyPath() throws Exception {
     
     @Test
     public void saveCourseConfiguration_HappyPath() throws Exception {
-        Long irrlevantSectionId = 3000L;
+        Long irrelevantSectionId = 3000L;
         Integer expectedDefaultMinutesPerSession = 100;
         Integer expectedTotalClassMinutes = 1000;
         Boolean expectedSimpleAttendanceValue = true;
-        
-        mockMvc.perform(post("/courseConfiguration/"+irrlevantSectionId+"/save")
+
+        mockMvc.perform(post("/courseConfiguration/"+irrelevantSectionId+"/save")
                 .param("saveCourseConfiguration", "Save Course Configuration")
                 .param("defaultMinutesPerSession", String.valueOf(expectedDefaultMinutesPerSession))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "defaultMinutesPerSession" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)